nautobot/core/admin.py

```diff
@@ -60,3 +57,53 @@ class NautobotModelAdmin(ModelAdmin):
         models.ForeignKey: {"widget": widgets.StaticSelect2},
         models.ManyToManyField: {"widget": widgets.StaticSelect2Multiple},
     }
+
+
+class LogEntryAdmin(NautobotModelAdmin):
+    """ModelAdmin for Django admin log entries that is read-only."""
+
+    date_hierarchy = "action_time"
+
+    list_filter = ["user", "content_type", "action_flag"]
+
+    search_fields = ["object_repr", "change_message"]
+
+    list_display = [
+        "action_time",
+        "user",
+        "content_type",
+        "object_link",
+        "action_flag",
+    ]
+
+    def has_add_permission(self, request):
+        """Disable add permission."""
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        """Disable change permission."""
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        """Disable delete permission."""
+        return False
+
+    def has_view_permission(self, request, obj=None):
+        return request.user.is_superuser
+
+    def object_link(self, obj):
+        """Disable delete permission."""
+        if obj.action_flag == DELETION:
+            link = escape(obj.object_repr)
+        else:
+            ct = obj.content_type
+            change_link = reverse(f"admin:{ct.app_label}_{ct.model}_change", args=[obj.object_id])
```

Comment:
Any chance we need to handle a NoReverseMatch here? Maybe if there's an object that has `create/delete` admin views but no `change` view?

Reply:
This is specifically for `admin` namespace, so all of those views/routes are automatically generated when a model is registered in the admin site. So it's not a concern IMO.

Reply:
What about install/uninstall apps case? (App installed with a ModelAdmin, log entries are created for it, then app is uninstalled)

Reply:
I've never had an issue in the admin panel for that. I'm guessing since the object would no longer be registered in the admin, as it's not installed, it wouldn't even show.